<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabOcr</class>
 <widget class="QWidget" name="TabOcr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">TabOcr</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QPushButton" name="openWizardButton">
     <property name="text">
      <string>openWizardButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>deviceSelector.title</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="FileSelector" name="deviceFileSelector" native="true"/>
      </item>
      <item>
       <widget class="DevicesComboBox" name="deviceComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>tesseractSelector.title</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="FileSelector" name="tesseractFileSelector" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>ocr.title</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>ocr.queue.title</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="ocr_addImageButton">
           <property name="text">
            <string>ocr.queue.addImageButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ocr_removeSelectedButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>ocr.queue.removeSelected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ocr_removeAllButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>ocr.queue.removeAll</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ocr_startButton">
           <property name="text">
            <string>ocr.queue.startOcrButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>ocr.results</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="ocr_acceptSelectedButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>ocr.results.acceptSelectedButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ocr_acceptAllButton">
           <property name="text">
            <string>ocr.results.acceptAllButton</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="ocr_ignoreValidateCheckBox">
           <property name="text">
            <string>ocr.results.ignoreValidate</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSelector</class>
   <extends>QWidget</extends>
   <header>ui.implements.components</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DevicesComboBox</class>
   <extends>QComboBox</extends>
   <header>ui.implements.components</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
